### PR TITLE
Use github.com instead of api.symfony.com

### PR DESCRIPTION
**Pull request by wouterj**
ref https://github.com/symfony/symfony-docs/issues/16322

### CODE REVIEW

**Review by fabpot — APPROVED** [review]

**Review by OskarStark — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Thank you Wouter.

File: src/Reference/MethodReference.php [hunk 1]
```diff
@@ -42,7 +42,7 @@ public function resolve(Environment $environment, string $data): ResolvedReferen
         return new ResolvedReference(
             $environment->getCurrentFileName(),
             $methodName.'()',
-            sprintf('%s/%s.html#method_%s', $this->symfonyApiUrl, str_replace('\\', '/', $className), $methodName),
+            sprintf('%s/%s.php#method_%s', $this->githubUrl, str_replace('\\', '/', $className), $methodName),
```
Comment (stof) [inline src/Reference/MethodReference.php:45]: these is no `method_...` id on GitHub.

Comment (wouterj) [reply src/Reference/MethodReference.php:45]: I know, here is me hoping they'll add this one day :roll_eyes:

Comment (OskarStark) [reply src/Reference/MethodReference.php:45]: I think they did sth. else for other languages already 🤔

Comment (stof) [reply src/Reference/MethodReference.php:45]: Well, their own code navigation logic is based on server-side knowledge of the lines are which you need to navigate. The generated links are using their `L*` anchors to navigate to specific line numbers. There is no special anchors in the page for the methods.

Comment (wouterj) [reply src/Reference/MethodReference.php:45]: If I'm correct, @OskarStark suggested them to add named anchors (i.e. class names or method names) besides line nr anchors and they were quite positive about that idea.

Comment (stof) [reply src/Reference/MethodReference.php:45]: Well, they haven't done it yet, then. And we don't know which naming pattern they will use if they finally do it.

Comment (OskarStark) [reply src/Reference/MethodReference.php:45]: Yes I did, but it's long time ago...

**Pull request merged**